test/format/format_config.c

```diff
@@ -1449,6 +1451,90 @@ config_statistics(void)
     }
 }
 
+/*
+ * config_chunk_cache --
+ *     Chunk cache configuration.
+ */
+static void
+config_chunk_cache(void)
+{
+    char buf[128];
+    const char *chunkcache_type;
+    const char *storage_source;
+
+    storage_source = GVS(TIERED_STORAGE_STORAGE_SOURCE);
+    chunkcache_type = NULL;
+
+    /* Chunkcache does not work unless tiered storage is configured with dir_store. */
+    if (!g.tiered_storage_config || strcmp(storage_source, "dir_store") != 0) {
+        if (config_explicit(NULL, "chunk_cache") && GV(CHUNK_CACHE))
+            testutil_die(EINVAL,
+              "%s: chunkcache cannot be enabled unless tiered storage is configured with dir_store",
+              progname);
+        return;
+    }
+
+    if (!config_explicit(NULL, "chunk_cache")) {
+        /*
+         * Make sure no configurations related to chunk caching are set if chunkcache is not
+         * enabled.
+         */
+        if (config_explicit(NULL, "chunk_cache.capacity") ||
+          config_explicit(NULL, "chunk_cache.chunk_size") ||
+          config_explicit(NULL, "chunk_cache.type") ||
+          config_explicit(NULL, "chunk_cache.storage_path"))
+            testutil_die(EINVAL,
+              "%s: Enable chunk caching (chunk_cache=on) to allow configuring other chunk cache "
+              "settings",
+              progname);
+
+        /* Enable chunkcache 50% of the time if not explicit set. */
+        testutil_snprintf(
+          buf, sizeof(buf), "chunk_cache=%s", mmrand(&g.data_rnd, 1, 100) <= 100 ? "on" : "off");
```

Comment:
Shouldn't this be `<= 50`?

Reply:
Yep, forgot to update it (added for testing purposes), updated in [b23ede9](https://github.com/wiredtiger/wiredtiger/pull/9589/commits/b23ede907b04cbd04fe0ee37772262b718a26e35)